### PR TITLE
fix: timezone modal design issues

### PR DESCRIPTION
**Pull request by sshanzel**
### Describe what this PR does
- Short and concise, bullet points can help
- Screenshots if applicable can also help

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-500 #done
WT-501 #done
WT-502 #done
WT-503 #done

### CODE REVIEW

File: packages/shared/src/components/auth/IncompleteRegistrationModal.tsx [hunk 1]
```diff
@@ -49,13 +49,14 @@ function IncompleteRegistrationModal({
       {...props}
       onRequestClose={onRequestClose}
       contentClassName="flex flex-col"
+      style={{ content: { overflow: 'hidden' } }}
     >
       <header className="flex relative justify-between items-center p-3 w-full border-b border-theme-divider-tertiary">
-        <span className="font-bold typo-body">Set your timezone</span>
+        <span className="ml-2 font-bold typo-body">Set your timezone</span>
```
Comment (rebelchris) [inline packages/shared/src/components/auth/IncompleteRegistrationModal.tsx:55]: I think we are stilling missing the height of the header to be 50px not?

**Review by rebelchris — CHANGES_REQUESTED** [review]
Think we're only missing the height still, but maybe I missed it in a previous one.

File: packages/shared/src/components/auth/IncompleteRegistrationModal.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/components/auth/IncompleteRegistrationModal.tsx:55]: Ah, nice catch. I totally forgot about which one to make it 50px. It is the height indeed.

**Review by rebelchris — APPROVED** [review]
Wohoo

**Pull request merged**